src/lib/widget/_utilities/api.utilities.ts

```diff
@@ -131,45 +136,53 @@ export function widgetApiFactory(componentApi: ReturnType<typeof _componentApi>)
             user: $userStore,
           });
 
-          if ($journeyStore) {
-            if ($journeyStore.successful && $oauthStore.successful && $userStore.completed) {
+          if ($oauthStore.error || $userStore.error) {
+            // If we get any errors from the stores, close the modal
+            formFactor === 'modal' && componentApi.close({ reason: 'auto' });
+          }
```

Comment:
If we have an OAuth or userinfo error, we close the modal and report the issue, since these can't be fixed by the user authenticating.